packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/utils.ts

```diff
@@ -0,0 +1,127 @@
+import { UniqueIdentifier } from "@dnd-kit/core";
+import { arrayMove } from "@dnd-kit/sortable";
+import { groupBy } from "lodash";
+import { AccountPage } from "../../../../components/hooks/useAccountPages";
+
+export interface PageList {
+  treeList: TreeElement[];
+  flatList: TreeElement[];
+}
+
+export interface TreeElement extends AccountPage {
+  parentId: string;
+  depth: number;
+  index: number;
+  expanded: boolean;
+  expandable: boolean;
+  collapsed: boolean;
+  children?: TreeElement[];
+}
```

Comment:
Feels to me like a type of something like this would be cleaner? 
```suggestion
export interface TreeElement {
  page: AccountPage;
  parentId: string;
  depth: number;
  index: number;
  expanded: boolean;
  expandable: boolean;
  collapsed: boolean;
  children?: TreeElement[];
}
```

Reply:
Followed this suggestion in 3366f44744c15d79b38e7446fe7cbbe42bf39b13